registry/slices/functions.go

```diff
@@ -635,9 +635,9 @@ func (sr *SlicesRegistry) Rest(list any) ([]any, error) {
 //	[]any - the list without the last element.
 //	error - error if the list is nil or not a slice/array.
 //
-// Example:
+// For an example of this function in a go template, refer to [Sprout Documentation: initial].
```

Comment:
Here and everywhere:

Go should be mentioned as a brand here, no?

```suggestion
// For an example of this function in a Go template, refer to [Sprout Documentation: initial].
```